gravitee-apim-console-webui/src/management/api/api-navigation/api-v4-menu.service.ts

```diff
@@ -35,10 +35,12 @@ export class ApiV4MenuService implements ApiMenuService {
     groupItems: MenuGroupItem[];
   } {
     const hasTcpListeners = api.listeners.find((listener) => listener.type === 'TCP') != null;
+    const isHttpProxyApi = api.type === 'PROXY' && !hasTcpListeners;
```

Comment:
it would be good to align the naming with the previous const
we should choose hasXxxListeners or isXxxxApi